### DISCUSSION TITLE
Terminal separators

### DISCUSSION THREAD

**Discussion by SchiSchi2** [Eugeny/tabby#3011] [General]
Hello all,

is there a way to make the window separators visible?
I mean the separators between the different consoles inside the terminus window.

Regards
SchiSchi

**Comment by kross-aeg** [comment]
+1
I've been wondering if this is possible too. Even a slight line would be helpful to separate panes. For vertically stacked panes, it sometimes looks like it's one tall pane if they are both full of logs.
